Gopkg.lock

```diff
@@ -36,6 +73,26 @@
   pruneopts = "UT"
   revision = "4dd4911251929a0ac4c4f3d8c4bda482a9e09f30"
 
+[[projects]]
+  branch = "master"
+  digest = "1:9c94a9509b279c5d406688d39bcd86f59e66c91e6c54081501dce4ce664383c8"
+  name = "github.com/giantswarm/e2e-harness"
```

Comment:
Please vendor deps once more. When I run `dep ensure -update` this one and few other deps get actually removed.

Reply:
@sslavic Thanks done. PTAL